backend/user-service/src/main/resources/requests/UserController/app-user-DELETE.http

```diff
@@ -0,0 +1,8 @@
+###
+DELETE http://localhost:8081/api/users/6c84fb95-12c4-11ec-82a8-0242ac130003
+
+###
```

Comment:
Tutaj jednak nie działa ten endpoint, wyrzuca 500, ale to raczej problem z forma user_db.